master/variables.tf

```
@@ -63,3 +79,15 @@ variable "tags" {
   description = "Tags to apply to all AWS resources created."
   type        = map(string)
 }
+
+variable "write_lambda_bucket_role_description" {
+  default     = "Allows sufficient permissions to write to the bucket that contains Lambda deployment packages] in the Master account."
```

Remove errant keystroke here:
```suggestion
  default     = "Allows sufficient permissions to write to the bucket that contains Lambda deployment packages in the Master account."
```

---

Please see commit 3aca79d.